src/runtime/node/readline/index.ts

```diff
@@ -0,0 +1,28 @@
+// https://nodejs.org/api/readline.html#readlineclearlinestream-dir-callback
+import type readline from "node:readline";
+import noop from "src/runtime/mock/noop";
```

Comment:
shouldn't this be a relative path? bare/absolute import like this would require build magic, which doesn't seem worth it.